bl-content/databases/plugins/about/db.php

```diff
@@ -0,0 +1,6 @@
+<?php defined('BLUDIT') or die('Bludit CMS.'); ?>
+{
+    "position": 1,
+    "label": "About",
+    "text": "We at Decrypt aim to write the best stuff"
+}
```

Comment:
Why is the auto generated database required for these changes ? :) These are static contents right ?